packages/oats/src/generate-types.ts

```diff
@@ -1321,7 +1169,7 @@ export function run(options: Options) {
                 undefined,
                 [],
                 undefined,
-                ts.createBlock([ts.createReturn(generateMakerExpression(schema))])
+                ts.createBlock([ts.createReturn(generateReflectionMaker(key))])
```

Comment:
Might be able to get rid of the `build*` fields entirely. I don't think they have a use outside the generated files.
We could move the `oar.make.createMaker(function () { return ...; })` so that `fromReflection` already returns that maker, and then tack that directly inside the `make` method of the classes. Something like this:

```ts
    static make(value: ShapeOfPhoto, opts?: oar.make.MakeOptions): oar.make.Make<Photo> {
      const make = oar.fromReflection(typePhoto.definition)(value, opts);
      return make.isError() ? oar.make.Make.error(make.errors) : oar.make.Make.ok(new Photo(make.success(), { unSafeSet: true }));
    }
```

Reply:
getting rid of some of the code gen would be nice.